Formula/t/tenv.rb

```diff
@@ -21,12 +21,12 @@ class Tenv < Formula
   conflicts_with "opentofu", because: "both install tofu binary"
   conflicts_with "terraform", because: "both install terraform binary"
   conflicts_with "terragrunt", because: "both install terragrunt binary"
-  conflicts_with "tfenv", because: "tfenv symlinks terraform binaries"
-  conflicts_with "tgenv", because: "tgenv symlinks terragrunt binaries"
+  conflicts_with "atmos", because: "both install atmos binary"
 
   def install
+    ENV["CGO_ENABLED"] = "0"
```

Comment:
what is the issue of enabling cgo?

Reply:
tenv is supported by many platforms like Linux / Windows or Macos. In case of Linux, it's not possible to use CGO_ENABLED for Alpine. So, we decided to disable it for all platforms.

Reply:
Homebrew doesn't build on Alpine for many reasons, so that shouldn't be an issue

Reply:
Yes, you're right. We just use the same flags for all builds: Linux / Windows / MacOs (https://github.com/tofuutils/tenv/blob/main/.goreleaser.yml#L20) and would like to keep it here too